doc/https.rst

```diff
@@ -0,0 +1,134 @@
+Secure with HTTPS
+=================
+
+To enable HTTPS on your BinderHub you can setup an ingress proxy and configure
+it to serve both, the Binder and JupyterHub interface, using TLS. You can
+either manually provide TLS certificates or use
+`Let's Encrypt <https://letsencrypt.org/>`_ to automatically get signed
+certificates.
+
+Setup IP & domain
+-----------------
+
+1. Get a static IP(v4) address that you will assign to your ingress proxy
+   later. For example, on Google Cloud this can be done using
+   ``gcloud compute addresses create <alias-name-for-ip> --region <region>``
+   and retrieve the assigned IP using ``gcloud compute addresses list``.
+2. Buy a domain name from a registrar. Pick whichever one you want.
+3. Set A records to your above retrieved external IP, one for Binder and
+   one for JupyterHub. We need two distinct subdomains for the routing to
+   the two different services as they will be served by the same proxy.
```

Comment:
Should we use ingress here just to use a word that will be used again later? I think the super mega correct and jargon'y word would be reverse proxy (not sure this would add to anything but confusion).

Reply:
I've put `ingress proxy` in the text, reverse would be the correct naming probably but this is not occurring in any other line yet.